README.rst

```diff
@@ -6,10 +6,63 @@ Django eduNEXT Audit Model
 Installation
 ############
 
+1. Install eox-audit-model:
+
+.. code-block:: python
+
+  pip install eox-audit-model
+
+2. Add “eox_audit_model” to your INSTALLED_APPS:
+
+.. code-block:: python
+
+  INSTALLED_APPS = [
+          ...
+         'eox_audit_model',
+  ]
+
+3. Run Migrate:
+
+.. code-block:: python
+
+  python manage.py migrate
+
+
 Usage
 #####
+Audit any execution of a method or function. This will create a database register with the following information:
+
+1. Status. If the process was successful or not.
+2. Action. The string given to identify the process.
+3. Time stamp. The execute date.
+4. Method name. Method or function name.
+5. Traceback log. If there was an exception, this will contain the traceback.
```

Comment:
Does this have to be a traceback log? I mean could we log the stdout when the status is success?

Reply:
do you mean the normal django logs ?

Reply:
yes, exactly. That or the output of `print()`